app/actions/add-new-goal.js

```diff
@@ -1,10 +1,30 @@
-import { INPUT_GOAL } from './../action_types.js';
+import * as types from './../action_types.js';
```

Comment:
Does this make sense?

This is just a stylistic point so I am not requesting changes